.travis.yml

```diff
@@ -10,6 +10,7 @@ addons:
       - libatlas-base-dev
       - liblapack-dev
       - gfortran
+      - libgfortran3
```

Comment:
I found out that you do not need to add libgfortran3. 
Actually I made already a PR regarding this issue and a little bit more. Please see here https://github.com/NeuralEnsemble/elephant/pull/57 . 
I would suggest to revert this and the change in the install script, after that I can merge.